src/blocks/cards.css

```diff
@@ -0,0 +1,52 @@
+.card {
+  display: flex;
+  flex-wrap: wrap;
+
+  background: #fff;
+  border: 1px solid #f3f3f3;
+
+  border-radius: 5px;
+  width: 200px;
+  height: 406px;
+}
+
+.card__content {
+  display: flex;
+  flex-direction: column;
+}
+
+.card__page {
+  margin: 32px 19px 40px;
```

Comment:
Don't add margins to each card element, you can add paddings 32px 16px for whole card instead and center content inside card using justify content and align-items rules